### DISCUSSION TITLE
How to disable the WebView Debugging features

### DISCUSSION THREAD

**Discussion by proneon267** [beeware/toga#1793] [Q&A] [answered]
In ``toga.WebView()``, is there any way to disable debugging features like `inspect element` or preventing the ``devTools window`` from opening up when ``F12`` is pressed?

**Comment by freakboy3742** [comment, marked as answer]
There's no built-in API at present to disable this popup at present.

You might be able to implement this functionality entirely in the web content; if your web page captures right click/F12 events, you can prevent the menu that allows the user to select "inspect element". This is how Google Docs and other web apps implement this functionality.

**Comment by proneon267** [comment]
Yeah, it seems that js capturing is the solution. Thanks for helping :)
